Formula/a/azure-core-cpp.rb

```diff
@@ -0,0 +1,42 @@
+class AzureCoreCpp < Formula
```

Comment:
Not too sure on name:
* Upstream maintains Vcpkg which is named `azure-core-cpp` - https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/vcpkg/vcpkg.json
* Also name used in https://anaconda.org/conda-forge/azure-core-cpp
* Though Gentoo went with `azure-core` - https://packages.gentoo.org/packages/dev-cpp/azure-core

Reply:
I think azure-core-app is better.